Cargo.toml

```diff
@@ -9,7 +9,7 @@ incremental = false
 
 # Disabling debug info speeds up builds a bunch,
 # and we don't rely on it for debugging that much.
-debug = 0
+debug = 1
```

Comment:
was this intended to be committed?

Reply:
Not really I think. Although with incremental disabled we might turn this on?